config/config.go

```diff
@@ -99,6 +99,7 @@ type ClientDefinition struct {
 	ResourceIDs          string `yaml:"resource_ids"`
 	Scopes               string `yaml:"scopes"`
 	Name                 string `yaml:"name"`
+	AllowPublic          string `yaml:"allowpublic"`
```

Comment:
this feels like it should probably be a boolean given the name, and the fact we assign it's value to "true" in the test below

Reply:
I set this as allowpublic to be consistent with the UAA docs https://docs.cloudfoundry.org/api/uaa/version/76.6.0/index.html#create-6

I actually agree with you, and with the field was instead 'allow_public'!

Reply:
Agreed 21d083ad3338bea0fc3eecbd81e82aeac968cf51